scripts/test/lint.py

```diff
@@ -273,6 +273,38 @@ def check(self, path):
         folder = path.parts[-2]
 
         return not (pkg_id == folder == nuspec[:-len(".nuspec")])
+        
+        
+        
+class UsesInvalidCategory(Lint):
+    # Some nuspec packages don't have a category
```

Comment:
I think some more details about why these packages don't have a category would be useful:
```suggestion
    # The common.vm, debloat.vm, and installer.vm packages are special as they
    # assist with the installation and allow to share code between packages.
    # They do not install any tool and consequently don't have a category.
```

Reply:
@sara-rn you have marked some feedback as resolved that still applies, have you maybe forgotten to push the changes it?